.travis.yml

```diff
@@ -9,3 +9,4 @@ script:
   # the "yast-travis-ruby" script is included in the base yastdevel/ruby image
   # see https://github.com/yast/docker-yast-ruby/blob/master/yast-travis-ruby
   - docker run -it -e TRAVIS=1 -e TRAVIS_JOB_ID="$TRAVIS_JOB_ID" yast-nfs-server-image yast-travis-ruby
```

Comment:
The `yast-travis-ruby` also runs `yardoc` by default. If you run it later again then it makes sense to disable that automatic call, that will save some time at Travis.

Use `yast-travis-ruby -x yardoc` option here, see e.g. https://github.com/yast/yast-packager/commit/465cdbce77c24f008b9e36653abf4dea86da6410

Reply:
Oh! Good to know. Thank you so much!